cpu/cc2538/dev/cc2538-rf.c

```diff
@@ -700,15 +732,15 @@ read(void *buf, unsigned short bufsize)
   }
 
   /* Read the RSSI and CRC/Corr bytes */
-  rssi = ((int8_t)REG(RFCORE_SFR_RFDATA)) - RSSI_OFFSET;
-  crc_corr = REG(RFCORE_SFR_RFDATA);
+  cc2538_last_rssi = ((int8_t)REG(RFCORE_SFR_RFDATA)) - RSSI_OFFSET;
+  cc2538_last_crc_corr_lqi = REG(RFCORE_SFR_RFDATA);
 
-  PRINTF("%02x%02x\n", (uint8_t)rssi, crc_corr);
+  PRINTF("%02x%02x\n", (uint8_t)cc2538_last_rssi, cc2538_last_crc_corr_lqi);
 
   /* MS bit CRC OK/Not OK, 7 LS Bits, Correlation value */
-  if(crc_corr & CRC_BIT_MASK) {
-    packetbuf_set_attr(PACKETBUF_ATTR_RSSI, rssi);
-    packetbuf_set_attr(PACKETBUF_ATTR_LINK_QUALITY, crc_corr & LQI_BIT_MASK);
+  if(cc2538_last_crc_corr_lqi & CRC_BIT_MASK) {
+    packetbuf_set_attr(PACKETBUF_ATTR_RSSI, cc2538_last_rssi);
+    packetbuf_set_attr(PACKETBUF_ATTR_LINK_QUALITY, cc2538_last_crc_corr_lqi & LQI_BIT_MASK);
```

Comment:
Yes, had you made them global you'd need to follow naming convention, but you have simply moved them to file scope (but still static), hence my request to undo the rename. Thank you!